app/screens/Login/index.js

```diff
@@ -13,6 +14,19 @@ class LoginContainer extends Component {
     error: null
   };
 
+  navListener;
```

Comment:
what is this?

Reply:
i forget assign this variable to listener. I want to save the instace of listener and then remove this when component `willUnmount`